Source/ASDisplayNode.mm

```diff
@@ -2915,10 +2915,6 @@ - (void)didEnterPreloadState
   ASDisplayNodeAssertLockUnownedByCurrentThread(__instanceLock__);
   [_interfaceStateDelegate didEnterPreloadState];
   
-  // Trigger a layout pass to ensure all subnodes have the correct size to preload their content.
-  // This is important for image nodes, as well as collection and table nodes.
-  [self layoutIfNeeded];
```

Comment:
If we remove `layoutIfNeeded` in this case would user with custom `ASDisplayNode` subclass that relied on the fact that the size was available via bounds / calculatedSize suddenly get an undefined behavior? 

They would have to basically do the same as `ASTableNode` and `ASCollectionNode` and explicitly allocating the view and trigger a layout if needed if they would like to have the old behavior?